packages/clay-panel/package.json

```diff
@@ -18,7 +18,7 @@
 		"build": "cross-env NODE_ENV=production babel src --root-mode upward --out-dir lib --extensions .ts,.tsx",
 		"buildTypes": "cross-env NODE_ENV=production tsc --project ./tsconfig.declarations.json",
 		"prepublishOnly": "yarn build && yarn buildTypes",
-		"test": "jest --config ../../jest.config.js"
+		"test": "jest --config ../../jest.config.js --updateSnapshot"
```

Comment:
You can remove `updateSnapshot` here, we don't want this command by default because it will make the snapshot unusable we should only use it when we intend to update the markup. If you maintain this, we can update in a snapshot in cases where the markup change is a bug.